modules/op-chains-generating-x509-secret.adoc

```diff
@@ -0,0 +1,55 @@
+// This module is included in the following assemblies:
+// * secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
```

Comment:
Not a big deal but just to be safe, remove the space so the mod doc tag is closer to the ID heading. Parsing AsciiDoc syntax on our portals can be delicate at times.